webapp/src/containers/Rhs/views/LinkedChannels/LinkedChannels.mock.ts

```diff
@@ -0,0 +1,219 @@
+export const mockLinkedChannels = [
+
+    {
+        msTeamsTeamID: 'string',
```

Comment:
Nitpicking a bit, but I feel like repeating the same string for all these risks missing bugs because arguments are passed in the wrong order since they all have the same value. For example, if we swap the team and channel name by mistake, the snapshot won't show any differences